packages/beacon-node/src/metrics/metrics/lodestar.ts

```diff
@@ -741,6 +742,11 @@ export function createLodestarMetrics(
         help: "Total number of imported blobs by source",
         labelNames: ["blobsSource"],
       }),
+      notOverrideFcuReason: register.gauge<{reason: NotReorgedReason}>({
```

Comment:
we should start using counter where applicable
```suggestion
      notOverrideFcuReason: register.counter<{reason: NotReorgedReason}>({
```